build/crds/devices/devices_v1alpha2_device.yaml

```diff
@@ -531,4 +531,4 @@ status:
     kind: ""
     plural: ""
   conditions: []
-  storedVersions: ["v1"]
+  storedVersions: [v1alpha2]
```

Comment:
maybe need a quote?  ["v11apha2"]

Reply:
not fixed, still the same error.

Reply:
OK, I will have a test on it.

Reply:
I guess whether it's related to the k8s version, On my local machine(k8s 1.22), I've not found this error. on the CI machine, it is kind 0.11 and is similar to k8s 1.21.

Reply:
Maybe try upgrading the Kind version, I will try to add kubebuilder comments to add storageversion like "// +kubebuilder:storageversion:v1alpha2"

Reply:
there're 2 e2e, 
`e2e` using `kubectl apply -f xxx.yaml` to create CRD, and it succeed.
keadm_e2e calls interface to create CRD and fails.

Reply:
I have tested the  `+kubebuilder:storageversion`  which have no effects.  
Run your code that have find two points:
-  the filepath read from "/etc/kubeedge/crds"
-  the update action will be failed with an error ""customresourcedefinitions.apiextensions.k8s.io \"devices.devices.kubeedge.io\" is invalid: metadata.resourceVersion: Invalid value: 0x0: must be specified for an update""